client/components/sticky-panel/style.scss

```diff
@@ -1,7 +1,10 @@
 .sticky-panel {
 	// Force a new block formatting context to prevent margin collapsing. It's needed for
 	// correct measurement of `.sticky_panel__content` height and setting the spacer height.
-	overflow: hidden;
+
+	.is-section-stats & {
```

Comment:
I wonder if we should add a comment here on why it's specific to Stats?

Reply:
Heh, if I only knew. I'll add something.